index.html

```diff
@@ -0,0 +1,168 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      rel="icon"
+      type="image/x-icon"
+      href="https://www.google.com/favicon.ico"
+    />
+    <link href="/src/output.css" rel="stylesheet" />
+    <title>Google</title>
+  </head>
+  <body>
+    <div
+      class="flex flex-col h-[111vh] md:h-[108vh] lg:h-[100vh] justify-between"
```

Comment:
Avoid using arbitrary values, instead, consider adding them in the `tailwind.config.js`. You can check the reason for not using the arbitrary `CSS` values inside the `HTML` file below. Even though it's not BEM classes we're using here, understand the concept behind not using them in HTML.

![image](https://user-images.githubusercontent.com/58233753/201971618-219d0ffa-d235-4d75-9ddf-0b1e8137d2c5.png)